docs/src/style.md

```diff
@@ -112,6 +112,37 @@ Bad:
 2(x + 1)
 ```
 
+#### Return statements
+
+Use an explicit `return` statement to exit from a function.  If the return from
```

Comment:
It helps to mention the motivation: without an explicit `return`, it could be confusing whether the author intended to return a certain value or not.